packages/fastify/src/withClerkMiddleware.ts

```diff
@@ -0,0 +1,62 @@
+import { parse } from 'cookie';
+import type { FastifyReply, FastifyRequest } from 'fastify';
+
+import { clerkClient } from './clerk';
+import { API_KEY, Cookies, FRONTEND_API, Headers, PUBLISHABLE_KEY, SECRET_KEY } from './constants';
+import { getSingleValueFromArrayHeader } from './utils';
+
+export const withClerkMiddleware = (options: Record<never, never>) => {
+  return async (req: FastifyRequest, reply: FastifyReply) => {
+    const cookies = parse(req.raw.headers.cookie || '');
+
+    // get auth state, check if we need to return an interstitial
+    const cookieToken = cookies[Cookies.Session];
```

Comment:
❓ Is this a different constant than the backend one? 

Edit: scratch that, just saw that you are simply re-exporting. Personally, I'd keep the `constants.` namespace instead of destructuring just to prevent confusing Cookies with a 3rd party package etc. Also, you can simply use the `constancts.` import for the other accessors (eg authorization header, origin etc)